src/schema.ts

```diff
@@ -131,44 +164,44 @@ export function build(data: any, opt: QueryConfig = {}): Schema {
   return new Schema(fieldSchemas);
 }
 
-export class Schema {
+export class Schema implements TableSchema {
```

Comment:
I believe its because `Schema` does indeed implement the required field of `TableSchema` - which is `fields`